coopiteasy_custom/models/analytic_account.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2020 Coop IT Easy SCRL fs
+#   Robin Keunen <robin@coopiteasy.be>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields, api
+from datetime import date, timedelta
+
+
+class AccountAnalyticAccount(models.Model):
+    _inherit = "account.analytic.account"
+    _order = "line_count desc"
+
+    line_count = fields.Integer(
+        string="Line Count", compute="_compute_line_count", store=True
+    )
+
+    @api.depends("line_ids", "account_type")
+    @api.multi
+    def _compute_line_count(self):
+        for account in self:
+            one_month_ago = date.today() - timedelta(days=30)
+            recent_lines = account.line_ids.filtered(
+                lambda l: fields.Date.from_string(l.date) >= one_month_ago
+            )
+            account.line_count = len(recent_lines)
```

Comment:
I'm wondering if it won't be easier to have one sql query for this as it could lead to some perf issue with time

Reply:
You're right. I won't fix but I'll leave a comment if we start having issues.